src/util-runmodes.c

```diff
@@ -192,8 +192,16 @@ int RunModeSetLiveCaptureAutoFp(ConfigIfaceParserFunc ConfigParser,
 
             threads_count = ModThreadsCount(aconf);
             for (thread = 0; thread < threads_count; thread++) {
+                char *printable_threadname = SCMalloc(sizeof(char) * (16+strlen(live_dev)));
```

Comment:
whats the 16 here?

Reply:
This was supposed to be an upper bound of the non-device-name part of the printable thread name, e.g the `W#01-`of `W#01-abcdefghijklmno`. In my next PR, I will also make this more dynamic.